test/functional/discovered_hosts_controller_test.rb

```diff
@@ -295,11 +295,11 @@ def test_no_dns_rebuild_if_dns_pending
     hostgroup = prepare_hostgroup_for_dns_rebuild(host)
     Nic::Managed.any_instance.expects(:rebuild_dns).never
     Host::Managed.any_instance.stubs(:skip_orchestration?).returns(false)
-    put :update, params: {:commit => "Update", :id => host.id,
```

Comment:
You no longer need to remove `commit` param, it is passed by default from some forms and is now ignored in core.